File: Python/tests/test_oms_ewmv.py
import pytest

from hbayesdm.models import oms_ewmv


def test_oms_ewmv():
    _ = oms_ewmv(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_oms_ewmv.R
context("Test oms_ewmv")
library(hBayesDM)

test_that("Test oms_ewmv", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(oms_ewmv(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
